### PR TITLE
Technopig/3.7.0 touchups

### PR DESCRIPTION
**Pull request by Technopig1992**
Corresponds to: 
#2118 

Closes:
#2111
#2073
#2070

### CODE REVIEW

**Comment by YakaryBovine** [comment]
The issue with Astromancer's Caster Upgrade Icon refers to this thing, not the research icon: 
image
However the research icon you added is good so we can leave that in.

**Comment by YakaryBovine** [comment]
image
How did the word "righteousness" get in here? The tooltip should indicate how much extra mana it consumes per target. "Replenish mana to up to 50 nearby friendly units. Each unit replenished will cost x additional mana."

File: mapdata/WarcraftLegacies/AbilityData/Skin/1163210817.json [hunk 1]
```diff
@@ -23,23 +23,18 @@
...
       "Id": 2020631137,
-      "Value": 2
+      "Value": 3
     },
     {
       "Level": 1,
       "Id": 829453409,
       "Type": 3,
-      "Value": "Necromantic Cleansing"
+      "Value": "Dispel"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1163210817.json:32]: Should be called "Dispel Magic" to match Priest version. Good change it from Necromantic Cleansing though.

**Comment by YakaryBovine** [comment]
image
These button positions for Necromancer don't look right. Also both Soul Burn and Dispel are on the E hotkey.

**Comment by YakaryBovine** [comment]
image
This quest should indicate that it lets Elven Runestones use Summon Mystic Defenders.

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
A handful of finishing touches are in order!

**Comment by Technopig1992** [comment]
> image
> 
> How did the word "righteousness" get in here? The tooltip should indicate how much extra mana it consumes per target. "Replenish 25 mana to up to 50 nearby friendly units. Each unit replenished will cost x additional mana."


Oh dear , must of been a very lazy copy and paste job, mb. Should the additional mana cost info be in the main body of text or in the secondary dark gray part ?

**Comment by Technopig1992** [comment]
> image
> 
> These button positions for Necromancer don't look right. Also both Soul Burn and Dispel are on the E hotkey.

This drove me crazy last night! I have changed the button Position of the raise dead ability to 0,2 multiple times but each time I test it's sitting right there in 1,1. I'll change the hot keys for dispel.

**Comment by Technopig1992** [comment]
> The issue with Astromancer's Caster Upgrade Icon refers to this thing, not the research icon: 
> 
> image
> 
> However the research icon you added is fantastic so we should leave that in. You don't have to fix the caster upgrade icon as part of this ticket, your call.

I'll pop it in n, one thing with the research icon, it's the same icon for both adept and master atm . Couldn't really find a fitting / matching one - may have to tweak the original a bit for the master.

**Comment by YakaryBovine** [comment]
> > The issue with Astromancer's Caster Upgrade Icon refers to this thing, not the research icon: 
> > 
> > image
> > 
> > However the research icon you added is fantastic so we should leave that in. You don't have to fix the caster upgrade icon as part of this ticket, your call.
> 
> I'll pop it in n, one thing with the research icon, it's the same icon for both adept and master atm . Couldn't really find a fitting / matching one - may have to tweak the original a bit for the master. 

Sweet as. Don't worry too much about Adept and Master being different; it would be nice but it's not a requirement.

**Comment by YakaryBovine** [comment]
> > image
> > 
> > These button positions for Necromancer don't look right. Also both Soul Burn and Dispel are on the E hotkey.
> 
> This drove me crazy last night! I have changed the button Position of the raise dead ability to 0,2 multiple times but each time I test it's sitting right there in 1,1. I'll change the hot keys for dispel. 
> 
> 

It's an auto cast ability so it has two button positions, one for off and one for on. Is that the issue?

**Comment by YakaryBovine** [comment]
> > image
> > 
> > How did the word "righteousness" get in here? The tooltip should indicate how much extra mana it consumes per target. "Replenish 25 mana to up to 50 nearby friendly units. Each unit replenished will cost x additional mana."
> 
> 
> Oh dear , must of been a very lazy copy and paste job, mb. Should the additional mana cost info be in the main body of text or in the secondary dark gray part ? 

As a general rule, just write down exactly what the spell does in plain English. Special formatting doesn't achieve much. So in this case, yeah all in the main body of the text.

**Comment by Technopig1992** [comment]
> > > image
> > > 
> > > These button positions for Necromancer don't look right. Also both Soul Burn and Dispel are on the E hotkey.
> > 
> > This drove me crazy last night! I have changed the button Position of the raise dead ability to 0,2 multiple times but each time I test it's sitting right there in 1,1. I'll change the hot keys for dispel. 
> > 
> > 
> 
> It's an auto cast ability so it has two button positions, one for off and one for on. Is that the issue?
> 

Ah I see , yeah didn't even occur to me. Will change

**Comment by YakaryBovine** [comment]
image
The Suppression debuff should indicate what it does, currently it just cuts off.

**Comment by YakaryBovine** [comment]
image
This buff icon should match the ability icon (except BTN rather than ATC).

**Comment by YakaryBovine** [comment]
Everything else is beautiful, nice job.

**Review by YakaryBovine — APPROVED** [review]
Good stuff, well done.

**Comment by Technopig1992** [comment]
ty
